tests/unit/client.py

```diff
@@ -106,6 +111,7 @@ def make_config():
 
         ])
         f.write(config)
+        f.close()
```

Comment:
This call is not required. when doing `with open....` it automatically closes the file when done. 
remove this line please.